src/styles/main.scss

```diff
@@ -1,3 +1,65 @@
+$calendar-width: 707px;
```

Comment:
Calendar width has to be calculated basing on calendar__day dimensions. You need to create variables for that.